2024-01-23 00:56:19 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2024-01-23 00:56:20 [main] INFO  c.a.t.t.s.u.GetMenuCategoriesTests$testThatDoesNotPolluteTheApplicationContextUris -
                Started GetMenuCategoriesTests.testThatDoesNotPolluteTheApplicationContextUris in 5.812 seconds (process running for 7.065)
2024-01-23 00:56:21 [http-nio-auto-1-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found with that UID. Please verify and try  again., uri=/api/owners-tools/customers/get-customer-uid/customerUid, timestamp=Tue, 23 Jan 2024 00:56:21 -0700}
2024-01-23 00:56:21 [http-nio-auto-1-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that uid found. Please check formatting and verify customer exists., uri=/api/owners-tools/customers/delete-customer/85, timestamp=Tue, 23 Jan 2024 00:56:21 -0700}
2024-01-23 00:56:21 [main] INFO  c.a.t.t.m.DbTests.NonEmptyTests -
                Starting NonEmptyTests using Java 17.0.6 with PID 120296 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:56:21 [main] DEBUG c.a.t.t.m.DbTests.NonEmptyTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:56:21 [main] INFO  c.a.t.t.m.DbTests.NonEmptyTests -
                No active profile set, falling back to 1 default profile: "default"
2024-01-23 00:56:22 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2024-01-23 00:56:22 [main] INFO  c.a.t.t.m.DbTests.NonEmptyTests -
                Started NonEmptyTests in 1.204 seconds (process running for 9.542)
2024-01-23 00:56:23 [http-nio-auto-1-exec-8] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=You have chosen a category that does not exist. Please check your spelling and formatting., uri=/api/menu/category, timestamp=Tue, 23 Jan 2024 00:56:23 -0700}
2024-01-23 00:56:23 [http-nio-auto-1-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that uid found. Please check formatting and verify customer exists., uri=/api/owners-tools/customers/edit-customer-phone/14/555.555.5156, timestamp=Tue, 23 Jan 2024 00:56:23 -0700}
2024-01-23 00:56:23 [main] INFO  c.a.t.t.mockTests.DbTests.EmptyTests -
                Starting EmptyTests using Java 17.0.6 with PID 120296 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:56:23 [main] DEBUG c.a.t.t.mockTests.DbTests.EmptyTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:56:23 [main] INFO  c.a.t.t.mockTests.DbTests.EmptyTests -
                No active profile set, falling back to 1 default profile: "default"
2024-01-23 00:56:24 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2024-01-23 00:56:24 [main] INFO  c.a.t.t.mockTests.DbTests.EmptyTests -
                Started EmptyTests in 1.381 seconds (process running for 11.524)
2024-01-23 00:56:24 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No orders found at all. Please contact your application team right away., uri=/api/owners-tools/orders/get-orders, timestamp=Tue, 23 Jan 2024 00:56:24 -0700}
2024-01-23 00:56:24 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customers found at all. Please contact your application team right away., uri=/api/owners-tools/customers/get-customers, timestamp=Tue, 23 Jan 2024 00:56:24 -0700}
2024-01-23 00:56:24 [http-nio-auto-1-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=You have chosen a menu item that does not exist., uri=/api/menu/id, timestamp=Tue, 23 Jan 2024 00:56:24 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-8] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order item not updated. Verify order item is part of order., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/88/2, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-10] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=We were unable to process your request. Please contact us directly when trying to order more than 10 of any given item., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/3/12, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order item not updated. Verify order exists., uri=/api/owners-tools/orders/update-order-item/77/3/2, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-9] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No order found with that UID. Please verify and try again., uri=/api/owners-tools/orders/get-order-uid/orderUid, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-1] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Can not delete order. Verify order exists., uri=/api/owners-tools/orders/delete-order/78, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:25 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found by that name. Please check your spelling and formatting., uri=/api/owners-tools/customers/get-customer-name/name, timestamp=Tue, 23 Jan 2024 00:56:25 -0700}
2024-01-23 00:56:26 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Menu item can not be added to order. Verify order id., uri=/api/owners-tools/orders/add-to-order/78/2/2, timestamp=Tue, 23 Jan 2024 00:56:26 -0700}
2024-01-23 00:56:26 [http-nio-auto-1-exec-9] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Unable to find com.adamstraub.tonsoftacos.entities.MenuItem with id 66, uri=/api/owners-tools/orders/add-to-order/654654-465465-555/66/2, timestamp=Tue, 23 Jan 2024 00:56:26 -0700}
2024-01-23 00:56:26 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No order found with that UID. Please verify and try again., uri=/api/owners-tools/orders/get-order-uid/orderUid, timestamp=Tue, 23 Jan 2024 00:56:26 -0700}
2024-01-23 00:56:26 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Starting GetOrdersByCustomerTests.testThatDoesNotPolluteTheApplicationContextUris using Java 17.0.6 with PID 120296 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:56:26 [main] DEBUG c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:56:26 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                No active profile set, falling back to 1 default profile: "default"
2024-01-23 00:56:27 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2024-01-23 00:56:27 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Started GetOrdersByCustomerTests.testThatDoesNotPolluteTheApplicationContextUris in 1.025 seconds (process running for 14.17)
2024-01-23 00:56:27 [http-nio-auto-4-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No orders for customer found., uri=/api/owners-tools/orders/get-order-customer/customer, timestamp=Tue, 23 Jan 2024 00:56:27 -0700}
2024-01-23 00:56:27 [http-nio-auto-4-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Session expired., uri=/api/owners-tools/customers/delete-customer/1, timestamp=Tue, 23 Jan 2024 00:56:27 -0700}
2024-01-23 00:56:27 [http-nio-auto-4-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=401 UNAUTHORIZED, message=User unauthorized., uri=/api/owners-tools/customers/delete-customer/1, timestamp=Tue, 23 Jan 2024 00:56:27 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-3] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Customer name incorrectly formatted. Please consult the documentation., uri=/api/owners-tools/customers/edit-customer-name/gd34-igjr/dsfk%20jh!@, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that id found., uri=/api/owners-tools/customers/edit-customer-name/54/Gary%20Garrison, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Order can not be closed if order is not ready., uri=/api/owners-tools/orders/close-order/654654-465465-555, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order cannot be found and as such can not be closed. Please verify order id., uri=/api/owners-tools/orders/close-order/66, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found with that UID. Please verify and try  again., uri=/api/owners-tools/customers/get-customer-uid/customerUid, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
2024-01-23 00:56:28 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order does not exist. Please verify order id and try again., uri=/api/owners-tools/orders/order-ready/88, timestamp=Tue, 23 Jan 2024 00:56:28 -0700}
